txnkv/txnsnapshot/scan.go

```diff
@@ -233,6 +233,7 @@ func (s *Scanner) getData(bo *retry.Backoffer) error {
 				ResourceGroupTag:  s.snapshot.mu.resourceGroupTag,
 				RequestSource:     s.snapshot.GetRequestSource(),
 				ResourceGroupName: s.snapshot.mu.resourceGroupName,
+				BusyThresholdMs:   uint32(s.snapshot.mu.busyThreshold.Milliseconds()),
```

Comment:
I don't understand why it is not protected by the RWLock?

Reply:
Sorry, I think the code before was actually incorrect. `Context` should not be set here. It should be set below in L243 instead.